src/components/add-tag-dialog/add-tag-dialog.html

```diff
@@ -9,14 +9,15 @@ <h2 class="mdc-dialog__header__title">
       <div class="add-tag-dialog__tag mdc-text-field mdc-text-field--upgraded mdc-text-field--fullwidth">
         <input type="text" class="mdc-text-field__input" placeholder="Tag">
         <div class="mdc-line-ripple"></div>
+        <p class="add-tag-dialog__error-text">You need to enter a non-repeating name for the tag</p>
       </div>
       <p class="mdc-text-field-helper-text" aria-hidden="true">
         Type tag name
       </p>
     </section>
     <footer class="mdc-dialog__footer">
       <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--cancel">Cancel</button>
-      <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--accept">OK</button>
+      <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--confirm">OK</button>
```

Comment:
You should not change mdc class here. It is better to remove mdc class here and add your own bem class